UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/wwwroot/js/rnw-admin.js

```diff
@@ -0,0 +1,30 @@
+﻿var warningTypeSelectedValue;
+var yeareSelectedValue;
```

Comment:
Please correct variable name as below, and update its all references.
```suggestion
var yearSelectedValue;
```

Reply:
renamed